quattor/types/hardware.pan

```diff
@@ -20,9 +20,12 @@ type structure_ram = {
 }
 type structure_cpu = {
     include structure_annotation
-    "speed"  : long # "CPU clock speed in MHz"
-    # Number of cores on each CPU chip, defaults to 1.
+    @{ desc = CPU clock speed in MHz }
+    "speed"  : long
+    @{ desc = Number of cores on each CPU chip }
     "cores" : long(1..) = 1
+    @{ desc = Number of execution threads on each CPU chip }
```

Comment:
hmmm, i don't to see how this is a property of a cpu. but i might be missing something. can you give an example of such config file (and in particular why this multiplier is coupled to the cpu and why the multiplier is not application specific).